tony-core/src/main/java/com/linkedin/tony/TonyApplicationMaster.java

```diff
@@ -670,6 +675,7 @@ private int doPreprocessingJob() throws Exception {
     }
 
     extraEnv.put(Constants.PREPROCESSING_JOB, "true");
+    extraEnv.put("HOME", System.getProperty("user.dir"));
```

Comment:
What is the purpose of setting `HOME`?

Reply:
Because YARN sets HOME to `/` and that is not the right home directory for our job. We don't see this problem since we normally don't access HOME. However.. Jupyter Notebook wants to write some cache files to $HOME and that results to permission error.

Reply:
Got it. Can you add a code comment about this?